.travis.yml

```diff
@@ -123,6 +123,7 @@ install:
 script:
 # run a basic sanity check to be sure that gatk doesn't explode
 # run tests
+# Because the docker build for GATK doesn't include the test code or the src files we have to manually collect them into a location to be mounted in for performing tests
```

Comment:
🤣

Reply:
I wish "travis script can't be parsed" showed up as a test failure on travis...